src/components/Pages/Unit/widgets/Recommended/Main/DisplayRecommended.js

```diff
@@ -34,11 +34,36 @@ export const renderPlaylistUnit = (unit, t, views = -1, watchingNow = -1, sugges
     <Table.Body>
       <Table.Row verticalAlign="middle">
         <Table.Cell textAlign="left" width={4}>
-          <UnitLogo
-            unitId={unit.id}
-            collectionId={getCollectionId(unit)}
-            fallbackImg='programs'
-          />
+          <div className="recommend-cell">
+            <UnitLogo
+              unitId={unit.id}
+              collectionId={getCollectionId(unit)}
+              fallbackImg='programs'
+            />
+            {(views !== -1 || watchingNow !== -1 || suggesterLabel) &&
```

Comment:
I believe this and the following conditions are correct, but that's not clear at all. Can you please do one of the following:

- move them to constants with clear names
- make html after condition to be a function with clear name

I'd prefer both, but that's up to you

Reply:
I understand that you just moved this block, but as you still understand what's going on here, can you make future life easier...

Reply:
Done.